enterprise/frontend/src/embedding-sdk/components/private/SdkPortalContainer.tsx

```diff
@@ -1,36 +1,24 @@
-import styled from "@emotion/styled";
-
 import ZIndex from "metabase/css/core/z-index.module.css";
-import {
-  EMBEDDING_SDK_FULL_PAGE_PORTAL_ROOT_ELEMENT_ID,
-  EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID,
-} from "metabase/embedding-sdk/config";
+import { EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID } from "metabase/embedding-sdk/config";
```

Comment:
The popovers/modals change are from https://github.com/metabase/metabase/pull/60431. I'll remove this once that PR has landed.